corehq/apps/case_search/templates/case_search/profile_case_search.html

```diff
@@ -62,7 +62,7 @@ <h3>Profile Slow Case Searches</h3>
       </li>
     </script>
 
-    <div class="panel-group" id="queries-accordion" data-bind="foreach: results().queries">
+    <div class="card-group" id="queries-accordion" data-bind="foreach: results().queries">
```

Comment:
Is this doing anything? I think `panel-group` is an HQ style, not a bootstrap style, and I don't see `card-group` in B5 docs.

Reply:
Hi Jenny, change `panel-group` to `card-group` seems have no affect. But here is the doc for `card-group`:https://getbootstrap.com/docs/5.0/components/card/#card-groups

Reply:
Cool, thank you. If it's not doing anything, let's leave it off. That class looks like it's meant for cards that will be displayed side-by-side, which we're not doing here.

Reply:
You're right. Sorry for the oversight. Changed here: becd3135baad122fa5641b250e948e123a81b047